### PR TITLE
Show all users and user profile

### PR DESCRIPTION
**Pull request by duyth1993**

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<li>
+	<%= gravatar_for user, size = 50 %>
```
Comment (duyth1993) [inline app/views/users/_user.html.erb]: indent

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class UsersController < ApplicationController
+  def index
+    @users = User.paginate(page: params[:page])
```
Comment (hnam2812) [inline app/controllers/users_controller.rb]: remove "()"

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -13,6 +17,10 @@ def create
     end
   end
 
+  def show
+    @user = User.find_by(id: params[:id])
```
Comment (hnam2812) [inline app/controllers/users_controller.rb]: remove "("

**Comment by hnam2812** [comment]
OK

File: app/helpers/users_helper.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module UsersHelper
+
+  def gravatar_for user, size = ""
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+    if !size.nil?
+      image_tag(gravatar_url, alt: user.name, class: "gravatar", style: "width: #{size}")
```
Comment (hellovietnam93) [inline app/helpers/users_helper.rb]: bỏ ()

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, @user.name) %>
```
Comment (hellovietnam93) [inline app/views/users/show.html.erb]: bỏ ()

**Comment by hellovietnam93** [comment]
OK

File: app/helpers/users_helper.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+module UsersHelper
+
+  def gravatar_for user, size = ""
+    gravatar_id = Digest::MD5::hexdigest user.email.downcase
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+    if !size.nil?
```
Comment (dieunb) [inline app/helpers/users_helper.rb]: unless đi em

**Pull request merged**